### ISSUE TITLE
planet subscriptions list does not recognize valid status, specifying a status it does recognize results in an error

### ISSUE THREAD

**Issue by jreiberkyle** [planetlabs/planet-client-python#815]
**Expected behavior**
```console
planet-client-python$> planet subscriptions list --limit 1 | jq .status
"running"
planet-client-python$> planet subscriptions list --status running
<a list of subscriptions with status: running>
```

**Actual behavior (describe the problem)**
```console
planet-client-python$> planet subscriptions list --limit 1 | jq .status
"running"
planet-client-python$> planet subscriptions list --status running
Usage: planet subscriptions list [OPTIONS]
Try 'planet subscriptions list --help' for help.

Error: Invalid value for '--status': 'running' is not one of 'created', 'queued', 'processing', 'failed', 'success'.
planet-client-python$> planet subscriptions list --status success
Error: {"error":{"reason":"Request failed schema validation","details":["parameter 'status' in query has an error: JSON value is not one of the allowed values"]}}
```

**Related Issues**
Please list the ticket numbers (e.g. #708) for any issues already logged in the system.

**Installation Method**
dev version

**Comment by jreiberkyle** [comment]
Looks like running isn't included in the CLI choice [here](https://github.com/planetlabs/planet-client-python/blob/main/planet/cli/subscriptions.py#L24). Soln is to add it. +1 is to move this from a CLI check to a api and cli check.

**Comment by kevinlacaille** [comment]
I'm on version 2.0.0-rc.3dev and I'm not able to replicate this bug. Namely, `planet subscriptions list --limit 1 --status running` just works for me.

Looks like "running" was added as a choice for `--status` about 3 months ago: https://github.com/planetlabs/planet-client-python/blob/main/planet/cli/subscriptions.py#L41

@jreiberkyle am I missing something?

**Comment by kevinlacaille** [comment]
Appears as though it was resolved in #821

**Comment by jreiberkyle** [comment]
yay my favorite type of issue - one that is already done!
